public/style_editor.js

```diff
@@ -8811,6 +8811,14 @@ function toggleSettingsPanel() {
 // Call the onPageLoad function when the page is loaded
 window.addEventListener('DOMContentLoaded', onPageLoad);
 
+let resizeTimeout;
+window.addEventListener('resize', () => {
+  clearTimeout(resizeTimeout);
+  resizeTimeout = setTimeout(() => {
+    location.reload();
```

Comment:
This seems like a very big hammer.
Doing it with a delay seems very hacky.
Can't we just resize the UI elements?

Reply:
Very quick, heavy handed fix, sorry.
Edited now, using flex boxes and real time resizing of the page.